ros/src/util/packages/autoware_launcher/documents/demos/rosbag.md

```diff
@@ -0,0 +1,36 @@
+# Rosbag Demo
+
+## Demo Data
+1. Download the following files from https://www.autoware.ai/ ([Mirror](https://drive.google.com/open?id=1ZwNQIJolJ2ogzpNprh89jCj6NocLi78f)).<br>
+  * https://www.autoware.ai/sample/sample_moriyama_data.tar.gz
+  * https://www.autoware.ai/sample/sample_moriyama_150324.tar.gz
+
+1. Make `~/.autoware` directory and extract the demo data.<br>
+```
+~/.autoware/data/tf/tf.launch
+~/.autoware/data/map/pointcloud_map/*.pcd
+~/.autoware/data/map/vector_map/*.csv
+~/.autoware/sample_moriyama_150324.bag
+```
+
+## Demo Run
+1. Start the Autoware Launcher.<br>
+```
+$ cd Autoware/ros
+$ ./run-experimental
+```
+![rosbag01](./images/rosbag01.png)
+1. Load a profile if needed.<br>
```

Comment:
1. If you have a previously created profile, you can load it selecting **Load Profile** from the Window/File menu.

Reply:
Applied your suggestion.